util/topgen.py

```diff
@@ -528,13 +528,24 @@ def generate_ac_range_check(topcfg: Dict[str, object], out_path: Path) -> None:
 
 
 # Generate RACL collateral
-def generate_racl(topcfg: Dict[str, object], out_path: Path) -> None:
+def generate_racl(topcfg: Dict[str, object], name_to_block: Dict[str, IpBlock],
+                  out_path: Path) -> None:
     # Not all tops use RACL
     if 'racl_config' not in topcfg:
         return
 
+    # Read the top-level RACL information
     topcfg['racl'] = parse_racl_config(topcfg['racl_config'])
 
+    # Generate the RACL mappings for all subscribing IPs
+    for m in topcfg['module']:
+        if 'racl_mappings' in m:
+            for if_name, mapping_path in m['racl_mappings'].items():
```

Comment:
Maybe `for if_name, mapping_path in m.get('racl_mappings', {}).items()`, which will avoid a layer of indentation.